test/views/declaration/TotalPackageQuantityViewSpec.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.declaration
+
+import base.Injector
+import forms.declaration.TotalPackageQuantity
+import models.Mode
+import org.jsoup.nodes.Document
+import play.api.i18n.MessagesApi
+import services.cache.ExportsTestData
+import unit.tools.Stubs
+import views.declaration.spec.UnitViewSpec
+import views.html.declaration.total_package_quantity
+
+class TotalPackageQuantityViewSpec extends UnitViewSpec with ExportsTestData with Stubs with Injector {
+
+  val template = new total_package_quantity(mainTemplate)
+
+  "Total Package Quantity view" when {
+
+    "rendered with empty form" should {
+      "have proper messages for keys" in {
+        val messages = instanceOf[MessagesApi].preferred(journeyRequest())
+        messages must haveTranslationFor("supplementary.totalPackageQuantity")
+        messages must haveTranslationFor("supplementary.totalPackageQuantity.empty")
+        messages must haveTranslationFor("supplementary.totalPackageQuantity.error")
+      }
+
+      onEveryDeclarationJourney { request =>
+        val view: Document = template.apply(Mode.Normal, TotalPackageQuantity.form())(request, messages)
+
+        "display same page title as header" in {
+          val viewWithMessage: Document = template.apply(Mode.Normal, TotalPackageQuantity.form())(request, realMessagesApi.preferred(request))
+          viewWithMessage.title() must include(viewWithMessage.getElementsByTag("h1").text())
+        }
+
+        "display section header" in {
+          view.getElementById("section-header").text() must include("supplementary.items")
+        }
+
+        "display header" in {
+          view.getElementById("title").text() must be("supplementary.valueOfItems")
+        }
+
+        "display empty input with label for Total Package" in {
+          view.getElementById("totalPackage-label").text() must be("supplementary.totalPackageQuantity") // FIXME
+//          view.getElementById("totalPackage-hint").text() must be("supplementary.totalPackageQuantity.hint") //FIXME there is no hint
```

Comment:
If there is no hint please remove this commented code

Reply:
Ok. This is rememberence for asking designers about copy here.